elements/bulbs-reading-list/components/bulbs-reading-list-menu.js

```diff
@@ -5,11 +5,6 @@ import {
 } from 'bulbs-elements/register';
 
 class BulbsReadingListMenu extends BulbsHTMLElement {
-  attachedCallback () {
-    invariant(this.getAttribute('reading-list-id'), '<bulbs-reading-list-menu> requires a "reading-list-id" attribute');
-    invariant(this.getAttribute('fixed-menu-min-width'), '<bulbs-reading-list-menu> requires a "fixed-menu-min-width" attribute');
-    invariant(this.getAttribute('pinned-container-selector'), '<bulbs-reading-list-menu> requires a "pinned-container-selector" attribute');
-  }
```

Comment:
None of this is needed now that the menu doesn't have an out of the box concept of sticking.